src/unstable/components/Form/story.js

```diff
@@ -62,7 +62,8 @@ const basicPokedexSchema = {
   properties: {
     Name: {
       type: 'string',
-      minLength: 5
+      minLength: 5,
+      examples: ['Pikachu', 'Snorlax', 'Charmander', 'Bulbasaur']
```

Comment:
Is this in the JSON schema spec?

Reply:
Yes, https://rjsf-team.github.io/react-jsonschema-form/

Reply:
@LucianBuzzo yes